app/Http/Requests/UpdateJobPoster.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use App\Services\Validation\Rules\DepartmentRule;
+use App\Services\Validation\Rules\ProvinceRule;
+use App\Services\Validation\Rules\SecurityClearanceRule;
+use App\Services\Validation\Rules\LanguageRequirementRule;
+use Illuminate\Support\Facades\Config;
+
+class UpdateJobPoster extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```

Comment:
Is this superseded by the authorizeResource call in the controller?

Reply:
I believe it is, but I'll confirm.

Reply:
We now have lots of options of where to run authorization: in the route file, in the controller, or in this Form Request. I liked having it all in the route file, so that all auth settings could be viewed in one place, but its the least flexible option. I've already had to break consistency and move some auth checks into controllers in some specific cases. Not sure what we should use going forward.

Reply:
Just confirmed, there's a test for this. `JobApiControllerTest->testUpdateAsWrongManager()`